book/src/accounts/authentication.md

```diff
@@ -35,8 +42,13 @@ environments due to the fact it is still possible to perform realtime phishing a
 
 ## Resetting Person Account Credentials
 
-Members of the `idm_account_manage_priv` group have the rights to manage person and service accounts
-security and login aspects. This includes resetting account credentials.
+Members of the groups `idm_people_admins`, `idm_people_on_boarding` and `idm_service_desk` have the
```

Comment:
Yeah, it's a new role with the intent that you can add service accounts to it which can create a person and do their credential reset, but otherwise can't edit or see the account at all.